backend/data/processors/conditions_preprocessing.py

```diff
@@ -405,13 +412,16 @@ def strip_bracket_spaces(processed):
 
 
 # -----------------------------------------------------------------------------
-# Phase 4: Common patterns
+# Phase 5: Common patterns
 # -------------------
 def uoc_in_business_school(processed):
     r"""Converts \d+UOC offered by the UNSW Business School to \d+UOC in F Business"""
     processed = re.sub(
         r"(\d+UOC) offered by the UNSW Business School", r"\1 in F Business", processed
     )
+    processed = re.sub(
+        r"(\d+UOC) of Business courses", r"\1 in ZBUS && COMM#", processed
```

Comment:
Why is the `&& COMM#` added here? The string being replaced doesn't seem to require the user to be enrolled in a certain program

Reply:
thanks for pointing this out, it was specifically for COMM3999 that required a commerce degree, i'll change this now